packages/redux/src/orders/actions/factories/fetchUserOrdersFactory.ts

```diff
@@ -19,18 +26,30 @@ import type { FetchOrdersAction } from '../../types/actions.types';
  */
 const fetchUserOrdersFactory =
   (getUserOrders: GetUserOrders) =>
-  (userId: number, query?: GetUserOrdersQuery, config?: Config) =>
-  async (dispatch: Dispatch<FetchOrdersAction>): Promise<Orders> => {
+  (userId: User['id'], query?: GetUserOrdersQuery, config?: Config) =>
+  async (dispatch: Dispatch<FetchOrdersAction>): Promise<OrderSummaries> => {
     try {
       dispatch({
         type: actionTypes.FETCH_USER_ORDERS_REQUEST,
```

Comment:
You will need to add the hash to the `meta` here as well as in the `FETCH_USER_ORDERS_FAILURE` action.